### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#55] [Announcements]
En hel massa förbättringar i den här versionen. Det blev mycket mer än jag tänkte mig från början.

För att nollställa dark mode så kan du shift-klicka kryssrutan. För att nollställa i Firefox så måste du shift-klicka kryssrutan, det fungerar inte att shift-klicka på texten.

Har du hittat en bugg? Har du något förslag? Kommentera här.

**Comment by nappen5** [comment]
Kanonbra. 5.1 ljud får jag ner med "UWTD", men inte med privatkopiera.

**Comment by matsnyst** [comment]
Fungerar perfekt, mina $5 donerade direkt.

**Comment by botanicblow** [comment]
Donerar gärna så snart det går att nyttja på denna typen av länk: https://www.svt.se/videoplayer-embed/29690610

**Reply by botanicblow** [reply]
Äh, donerar ändå 👍

**Reply by stefansundin** [reply]
Bra idé!

**Reply by stefansundin** [reply]
Detta stöds i v0.3.1. Tack för idén. f9a49c4a24c8d849e98adb605315113a45660f7a

**Comment by tomiisek** [comment]
"Undertexter inkluderas automatiskt. De finns inte i listan längre."

Tyvärr funkar inte, undertexter inte inkluderas.

**Reply by Lemuren01** [reply]
Stämmer.
Inga undertexter för mig heller.

Donerade en femma vilket fall som helst eftersom att jag tycker att detta tillägget är kanon i firefox.

**Reply by stefansundin** [reply]
Fixar nog det imorgon tror jag. För tillfället kan du manuellt ändra filändelsen till `.mkv` eller lägg till `-scodec mov_text`. Eller ändra filändelsen till `.srt` för att få ner en separat fil som förut.

**Comment by Virablatira** [comment]
Hej. Programmet är jättebra. Har donerat innan.

Men jag föredrar den tidigare funktionen att undertexter går att hämta separat.

**Reply by tomiisek** [reply]
Jag vill gärna att undertexter inkluderas automatiskt, men samtidigt att de finns kvar i listan, ibland behöver man endast undertexter utan video.

**Reply by stefansundin** [reply]
Har precis släppt v0.3.1 och där finns det i menyn igen samtidigt som det inkluderas automatiskt.

**Reply by Virablatira** [reply]
Toppen tack!
